automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -214,7 +214,7 @@ def product_upgrade(product):
                                 upgraded = str(execute(
                                     get_sat_cap_version, 'cap', host=cap_host
                                     )[cap_host])
-                                if current:
+                                if current != 'None':
```

Comment:
I feel if 'None' is a string then something is wrong here.
I feel we shouldn't be checking for `current != 'None'` this.